a3p-integration/proposals/a:upgrade-14/invite-submission/README.md

```diff
@@ -0,0 +1,9 @@
+The main core-eval submission here is the wallet upgrade, which is specified in
+package.json and built in ./submission.
```

Comment:
not true anymore but not a blocker.

The wallet upgrade is part of the upgrade-14 `"upgradeInfo"`